### PR TITLE
#164829196 setup heroku pipeline

### PR DESCRIPTION
**Pull request by vino-jasuba**
## What does this PR do?

adds necessary configuration for deployment on Heroku

## Description of task to be completed

Deploy base application to heroku using heroku pipelines

## How can this be manually tested?

Visit app url using HTTP client

### CODE REVIEW

File: authors/apps/authentication/urls.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
     LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
 )
 
+app_name = 'authentication'
+
 urlpatterns = [
```
Comment (ogiste) [inline authors/apps/authentication/urls.py:9]: We should use the 'path' function to set the various URLs on the application so as to follow the new Django standards

**Review by ogiste — COMMENTED** [review]
Update the URLs to use the path() function over the  old url() syntax

File: authors/apps/authentication/urls.py (hunk 1 above)
Comment (vino-jasuba) [reply authors/apps/authentication/urls.py:9]: I think that change was already made or is being handled on this [branch](https://github.com/andela/ah-centauri-backend/tree/ch-upgrade-dependencies-164829189)

@Sm0k3s

**Review by ValerieRono — COMMENTED** [review]
ensure application is deploying, 
you can use $ heroku logs --tail

**Comment by vino-jasuba** [comment]
> ensure application is deploying,
> you can use $ heroku logs --tail

@ValerieRono , the application deploys successfully. We just need to implement the `JWTAuthentication` 

This is the error that you will get when you try accessing the endpoints

`NotImplementedError at /api/users
.authenticate() must be overridden.`

**Comment by ValerieRono** [comment]
> > ensure application is deploying,
> > you can use $ heroku logs --tail
> 
> @ValerieRono , the application deploys successfully. We just need to implement the `JWTAuthentication`
> 
> This is the error that you will get when you try accessing the endpoints
> 
> `NotImplementedError at /api/users .authenticate() must be overridden.`

okay well noted

**Review by ValerieRono — APPROVED** [review]
looks good to me

**Review by ogiste — APPROVED** [review]
Looks good to me.

**Pull request merged**